### PR TITLE
Add header stats

### PR DESCRIPTION
**Pull request by Mykhailenko**

### CODE REVIEW

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java [hunk 1]
```diff
@@ -674,6 +678,16 @@ public void onSuccess(String result) {
         this.schedulerTimerUpdate.scheduleRepeating(SchedulerConfig.get().getClientRefreshTime());
     }
 
+    private static String getJsonStringNullable(JSONObject jsonObject, String attributeName, String defaultValue) {
+        JSONString result = jsonObject.get(attributeName).isString();
```
Comment (ShatalovYaroslav) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java]: Can you use Optional.orElse to make code more readable?

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java [hunk 2]
```diff
@@ -674,6 +678,16 @@ public void onSuccess(String result) {
         this.schedulerTimerUpdate.scheduleRepeating(SchedulerConfig.get().getClientRefreshTime());
     }
 
+    private static String getJsonStringNullable(JSONObject jsonObject, String attributeName, String defaultValue) {
```
Comment (pedrovelho) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java]: Comment for this function?

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java [hunk 3]
```diff
@@ -623,8 +623,25 @@ public void statusChanged(SchedulerStatus status) {
...
-        schedulerStatusLabel.setContents("Status:" + status.name());
+
+        String neededNodes = "0";
+        String pendingEligibleTasks = "0";
+        if (controller.getModel().getSchedulerStatistics() != null) {
+            neededNodes = controller.getModel().getSchedulerStatistics().get("NeededNodes");
+            pendingEligibleTasks = controller.getModel().getSchedulerStatistics().get("PendingEligibleTasks");
+        }
+
+        if (!Character.isDigit(neededNodes.charAt(0))) {
```
Comment (ShatalovYaroslav) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java]: Can you extract this repeated check to private method?

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java [hunk 4]
```diff
@@ -623,8 +623,25 @@ public void statusChanged(SchedulerStatus status) {
...
                 break;
         }
+
         // Update the scheduler status label
-        schedulerStatusLabel.setContents("Status:" + status.name());
+
+        String neededNodes = "0";
+        String pendingEligibleTasks = "0";
+        if (controller.getModel().getSchedulerStatistics() != null) {
+            neededNodes = controller.getModel().getSchedulerStatistics().get("NeededNodes");
```
Comment (ShatalovYaroslav) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java]: Maybe small optimization: only 1 call controller.getModel().getSchedulerStatistics(), if this call is expensive.

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java [hunk 5]
```diff
@@ -623,8 +623,25 @@ public void statusChanged(SchedulerStatus status) {
...
+            pendingEligibleTasks = controller.getModel().getSchedulerStatistics().get("PendingEligibleTasks");
```
Comment (ShatalovYaroslav) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java]: "NeededNodes" "PendingEligibleTasks" extract as constants of class?

**Pull request merged**